interfaces.go

```diff
@@ -449,6 +460,28 @@ func (e *Event) transactionMarshalJSON() ([]byte, error) {
 	return json.Marshal(x)
 }
 
+func (e *Event) checkInMarshalJSON() ([]byte, error) {
+	checkIn := serializedCheckIn{
+		CheckInID:     e.CheckInID,
+		MonitorSlug:   e.CheckIn.MonitorSlug,
+		Status:        e.CheckIn.Status,
+		Duration:      e.CheckIn.Duration,
+		Release:       e.Release,
+		Environment:   e.Environment,
+		MonitorConfig: nil,
+	}
+
+	if e.MonitorConfig != nil {
+		checkIn.MonitorConfig = &MonitorConfig{}
+		checkIn.MonitorConfig.Schedule = e.MonitorConfig.Schedule
+		checkIn.MonitorConfig.CheckInMargin = e.MonitorConfig.CheckInMargin
+		checkIn.MonitorConfig.MaxRuntime = e.MonitorConfig.MaxRuntime
+		checkIn.MonitorConfig.Timezone = e.MonitorConfig.Timezone
+	}
```

Comment:
Yes I know it looks bad. It's 10 PM on Tuesday and I can't think of anything better right now. So sorry 🙈